### PR TITLE
add verify via cosign

### PR DESCRIPTION
**Pull request by pxp928**

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -44,6 +44,36 @@ A video format of these setup instructions is available here:
 - [Docker](https://docs.docker.com/get-docker/)
 - [Docker Compose](https://docs.docker.com/compose/install/)
 - [jq](https://stedolan.github.io/jq/download/)
+- [cosign](https://docs.sigstore.dev/system_config/installation/)
+- [crane](https://github.com/google/go-containerregistry/blob/main/cmd/crane/README.md)
+
+## Step 0: Verify GUAC image via Cosign
+
+1. Based on the
```
[the remark this answers is not in the document]
Comment (pxp928) [reply setup.md:53]: done.

File: setup.md [hunk 2]
```diff
@@ -44,6 +44,37 @@ A video format of these setup instructions is available here:
 - [Docker](https://docs.docker.com/get-docker/)
 - [Docker Compose](https://docs.docker.com/compose/install/)
 - [jq](https://stedolan.github.io/jq/download/)
+- [cosign](https://docs.sigstore.dev/system_config/installation/)
```
Comment (pxp928) [inline setup.md:47]: moving to its own page

File: setup.md [hunk 3]
```diff
@@ -45,32 +45,37 @@ A video format of these setup instructions is available here:
...
+   [latest GUAC release](https://github.com/guacsec/guac/releases/latest) if you
+   have not already done so. For example Linux x86_64 is
    [`guacone-linux-amd64'](https://github.com/guacsec/guac/releases/latest/download/guacone-linux-amd64).
 
-1. Rename the binary to `guacone`, mark it executable if necessary, and add it
+2. Rename the binary to `guacone`, mark it executable if necessary, and add it
    to your shell's path.
 
-1. Download the compose files from the
+3. Download the compose files from the
```
Comment (jeffmendoza) [inline setup.md:64]: Does your editor do this?

**Review by jeffmendoza — APPROVED** [review]

File: setup.md (hunk 3 above)
Comment (pxp928) [reply setup.md:64]: yeah, not sure what caused the numbering to be weird...

Comment (pxp928) [reply setup.md:64]: ah that I did not know. Let me change it back

File: setup.md [hunk 4]
```diff
@@ -45,32 +45,37 @@ A video format of these setup instructions is available here:
 - [Docker Compose](https://docs.docker.com/compose/install/)
 - [jq](https://stedolan.github.io/jq/download/)
 
+## Requirements
+
+- Completion of the [Vertification of the GUAC images and
```
Comment (lumjjb) [inline setup.md:50]: should this be optional?

Comment (pxp928) [reply setup.md]: made it optional

**Review by lumjjb — APPROVED** [review]
just 1 comment, else LGTM

File: verification.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
+---
+layout: page
+title: Verify and validate via cosign and slsa-verifier
+permalink: /verification/
+parent: Getting started with GUAC
+nav_order: 1
```
[the remark this answers is not in the document]
Comment (pxp928) [reply verification.md:6]: As a software supply chain security focused project, feels like we should place this first (to show that we are producing the proper signed outputs with attestations) and that people should get in the habit of verifying before using.

**Pull request merged**